code/__HELPERS/game.dm

```diff
@@ -238,6 +238,9 @@
 
 ///Calls the show_candidate_poll_window() to all eligible ghosts
 /proc/poll_candidates(question, jobban_type, be_special_flag = 0, poll_time = 300, ignore_category = null, flashwindow = TRUE, list/group = null)
+	if (group.len == 0)
```

Comment:
```suggestion
	if (group?.len == 0)
```
The args show that this can be null

Reply:
`null` makes absolutely no sense as a value here and I would rather it runtime. I don't think the args show anything other than that `group` was added later on, and you can't have parameters after ones with a default. Notice how every caller uses `group = `, or passes them all.

Reply:
Sounds good to me